tests/output/pjs/output_test.js

```diff
@@ -145,7 +145,7 @@ describe("Scratchpad Output Exec", function() {
     });
 
     failingTest("Too Many Draw Operations", function() {
-        for (var i = 0; i < 17000; i++) {
+        for (var i = 0; i < 1000000; i++) {
```

Comment:
When testing locally 17000 wasn't enough to trigger the loop protector so this test was intermittently failing on travis-ci b/c although the loop protect code was being triggered, mocha was timing out b/c both timeouts were 2000ms.  This is enough to trigger it in both places.